src/runtime/virtcontainers/sandbox.go

```diff
@@ -1596,6 +1596,11 @@ func (s *Sandbox) CreateContainer(ctx context.Context, contConfig ContainerConfi
 
 			logger.Debug("Removing stopped container from sandbox store")
 			s.removeContainer(c.id)
+			// flush data to storage, cleanup removed container
+			// here can ignore the error
+			if e := s.Save(); e != nil {
```

Comment:
nit: `errSave` to be consistent with code above.

Reply:
done :)